packages/apps/sales/src/components/atoms/ProductCard/ProductCard.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useMemo} from 'react';
+import {StyleSheet} from 'react-native';
+import {
+  ObjectCard,
+  TextUnit,
+  usePriceFormat,
+  useThemeColor,
+} from '@axelor/aos-mobile-ui';
+import {useMetafileUri} from '@axelor/aos-mobile-core';
+
+export interface ProductCardProps {
+  style?: any;
+  onPress: () => void;
+  picture: any;
+  name: string;
+  code: string;
+  productFamily: string;
+  productCategory: string;
+  description: string;
+  configuratorIsNull: boolean;
+  unit: string;
+  salePrice: number;
+}
+
+const ProductCard = ({
+  style,
+  onPress,
+  picture,
+  name,
+  code,
+  productFamily,
+  productCategory,
+  description,
+  configuratorIsNull,
+  unit,
+  salePrice,
+}: ProductCardProps) => {
+  const Colors = useThemeColor();
+  const formatMetaFile = useMetafileUri();
+  const priceFormat = usePriceFormat();
+
+  const borderStyle = useMemo(() => {
+    if (configuratorIsNull) {
+      return getStyles(Colors.progressColor.background)?.border;
+    }
```

Comment:
All the logic around configuratorIsNull is good but the name is reversed, rename it isConfigurator or something like this